config/locales/en.yml

```diff
@@ -477,7 +477,7 @@ en:
       no_questions_results_found: "Hi! No Questions have this tag <span>&#x1F937</span>, try checking the <b>Research Notes</b> or <b>Wikis</b> tabs on this page."
       try_advanced_search: "Or try an <a href='%{url1}'>advanced search</a>"
       people: "People"
-      no_map_results_found: "Sorry, no map is tagged with <i>%{tag}</i>"
+      no_map_results_found: "Sorry, no map is tagged with <i>%{tag}</i>. You can view other maps <a href="/maps">here</a>."
```

Comment:
Ahh, I see why the tests are failing. I think that this needs to be changed to single quotes. Sorry, that's my mistake!

@Vishal01-git, can you change this part to `You can view other maps <a href='/maps'>here</a>` instead?

Reply:
You can do this by pushing another commit to your branch `Vishal01-git:vishalFTO`. Let us know if you need help figuring this out!

Reply:
Ok sure i will push another commit with the changes.